initialize/services_test.go

```diff
@@ -0,0 +1,39 @@
+package initialize
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/quay/clair/v4/config"
+	"github.com/quay/claircore/libvuln"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLocalMatcherDisableUpdaters(t *testing.T) {
+	originalLibvulnNew := libvulnNew
+	defer func() {
+		libvulnNew = originalLibvulnNew
+	}()
+
+	libvulnNew = mockLibvulnNewDisableUpdaters
+	ctx := context.Background()
+	cfg := config.Config{
+		Matcher: config.Matcher{
+			DisableUpdaters: true,
+		},
+		Updaters: config.Updaters{
+			Sets: []string{"alpine", "aws"},
+		},
+	}
+
+	_, err := localMatcher(ctx, &cfg)
+	assert.NoError(t, err)
```

Comment:
It would be nice to avoid adding a new dependency for this line, nothing against testify.

(I know this is probably over-shared https://www.youtube.com/watch?v=PAAkCSZUG1c&t=568s&ab_channel=TheGoProgrammingLanguage)

Reply:
I agree with this - good thoughts. I hadn’t actually seen the video, he makes some good points!